### DISCUSSION TITLE
Porn Fetch V3.3

### DISCUSSION THREAD

**Discussion by EchterAlsFake** [EchterAlsFake/Porn_Fetch#38] [Announcements]
# Changelog

## New Features
- Porn Fetch supports discord rich presence (although it's optional and disabled by default)
- Porn Fetch now checks connection to all sites and lets you know if there was an error
- Added model support for xvideos
- You can now select a range of videos
- Added sorting to the tree widget
- Added searching support for files

## Bug fixes
- Fixed application tabbing
- Fixed the total progressbar showing no percent value after it is finished

## Other
- Huge code refactoring
- Added CI/CD build actions for all platforms
- Switched to Qt 6.7.0
- Improved terminal debug messages
- Reduced file size by over 70% on Windows and Linux
- Improved the tree widget
- Switched building to pyside6-deploy (nuitka)
- Improved visual appearance and startup time

## Deprecations
- removed all metadata functionality from Porn Fetch, because it's useless and hard to maintain.

# Android

> [!NOTE]
> Android support in Qt 6.7.0 is a bit broken. The UI looks terrible and it has some glitched artifacts, but the application itself works. If downloads don't start you need to wait or click on download a second time. I hope this will be fixed with Qt 6.7.1, but I don't want to compile the dev branch of Qt for PySide6 in hope of this being fixed as it would be toooooo much work.

Porn Fetch v3.3 has support for all Android architectures :)

# Downloads

GUI: The GUI is the graphical user interface of Porn Fetch. This is probably what you want
CLI: The CLI is a terminal version for experienced users,

x64: This is for modern computers and probably what you want
x32: This is for 32bit systems. Use this if you are on a really old system

> [!NOTE]
> The Anroid .apk package has different architectures. You need to find out which architecture your android device has. Here are some hints:

- aarch64: Newer devices like Xiaomi Redmi 10, Pixel 7 Pro etc.
- armv7: Really old devices like the Samsung Galaxy S4 mini or the Xiaomi Redmi 6
- x86: I don't even know this lol
- x86_64: This is mostly for desktop PCs using Android like ChromeOS or something idk lol.

(So you probably want aarch64)

<hr /><em>This discussion was created from the release <a href='https://github.com/EchterAlsFake/Porn_Fetch/releases/tag/3.3'>Porn Fetch V3.3</a>.</em>

**Comment by properlogic111** [comment]
Is it possible to have the thumbnails shown?

**Reply by EchterAlsFake** [reply]
Will be implemented in the next release
